dmutils/email.py

```diff
@@ -40,6 +43,7 @@ def send_email(to_email_addresses, email_body, api_key, subject, from_email, fro
             'track_clicks': False,
             'auto_text': True,
             'tags': tags,
+            'metadata': metadata,
```

Comment:
Is metadata just anything you want it to be, or does it have to be a specific thing (like a string or a dict or something else)?